### PR TITLE
Add bull and announce location

### PR DESCRIPTION
**Pull request by inesilva**
# Description
- It adds the bull to the page titles
- Announce location

## [🔗 Preview Link](https://deploy-preview-404--silly-leavitt-2e0668.netlify.app/)

### CODE REVIEW

**Comment by davidhajba** [comment]
hey @inesilva 
On my 16 inch screen I see this:
Screenshot 2023-06-07 at 16 30 26

would it be possible to make the 'logo+bull' responsive too, 
as the svg element in the background, 
so on all desktop screens see something like this ?

Screenshot 2023-06-07 at 16 31 09

**Review by oscard0m — CHANGES_REQUESTED** [review]
frontpage is missing the date and location TBA, at least in my screen on Desktop: 
image

@davidhajba could you take a look on mobile? LOCATION TBA gets an extra so I think we should consider a way to better arrange this.

File: _includes/hero.html [hunk 1]
```diff
@@ -4,8 +4,7 @@
       {{ site.data.event.date }} / Location {{ site.data.event.location }}
     </p>
     <div class="logos-container">
-      <img src="/images/logo_vertical.svg" alt="EmberFest logo" />
-      <img src="/images/glasses-big.webp" alt="red glasses" />
+      <img src="/images/title-with-bull.svg" alt="EmberFest logo with a bull underneath wearing red glasses" />
```
Comment (oscard0m) [inline _includes/hero.html:7]: Could you review the associated CSS? If `<div class="logos-container">` has one child, the `display: flex`, `gap`, etc do not apply anymore 🧹.

**Review by marcoow — CHANGES_REQUESTED** [review]
"LOCATION TBA" should be changed to "MADRID"

This doesn't work on mobile yet: 
deploy-preview-404--silly-leavitt-2e0668 netlify app_

**Comment by davidhajba** [comment]
`could you take a look on mobile`
- sure, on mobile it looks good to me

`it is responsive. i added a max-width of 650px which is the size I see on sketch. should it be bigger than this?`
- in sketch I used a 14 inch screen, on larger screens the 'logo+bull head' should grow bigger together with the background svg element, would it be possible ?

**Comment by oscard0m** [comment]
> * in sketch I used a 14 inch screen, on larger screens the 'logo+bull head' should grow bigger together with the background svg element, would it be possible ?

@davidhajba, just to be clear I'm understanding our request:
1. How the background resizes (in production for large screens) is OK **but** we need to adjust how the bull resizes for those screen sizes.
or
2. How the background resizes (in production for large screens) is NOT OK **and** we need to adjust how the bull resizes for those screen sizes.

**Comment by oscard0m** [comment]
> * in sketch I used a 14 inch screen, on larger screens the 'logo+bull head' should grow bigger together with the background svg element, would it be possible ?

Can you take a look into it now? @davidhajba

**Comment by davidhajba** [comment]
yep - looks perfect, thanks 👌

**Review by davidhajba — APPROVED** [review]

**Review by oscard0m — APPROVED** [review]

**Review by marcoow — APPROVED** [review]

**Pull request merged**